routes/user.js

```diff
@@ -0,0 +1,67 @@
+const express = require('express'),
+    passport = require('passport'),
+    ensureLoggedIn = require('connect-ensure-login').ensureLoggedIn(),
+    router = express.Router();
+
+/* GET user profile. */
+router.get('/', ensureLoggedIn, (req, res, next) => {
+    console.log(req.user);
+    res.render('user', {
+        user: req.user
+    });
+});
+
+router.get('/profile', ensureLoggedIn, (req, res, next) => {
```

Comment:
ten route odpowiada adresowi GET /user/profile.
W pliku profile.js masz metody które odpowiadają za obsługę GET/POST /profile.

Logika zawarta w tym pliku jak i w pliku `profile.js` wyglada podobnie i chyba wystarczy jedno z tych miejsc. Ja skorzystałabym z profile.js a tutaj usunelabym metody zwiazane z /user/profile

Jedyne co może być problemem to czy `froms` zwraca odpowiedni HTML w metodach success/error. Podsyłam dokumentację do forms, może okazać się pomocna - https://github.com/caolan/forms